experimental/packages/exporter-trace-otlp-proto/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@opentelemetry/exporter-trace-otlp-proto",
   "version": "0.27.0",
+  "private": "true",
```

Comment:
really?

Reply:
I am actually not sure. Right now one possibility was to deploy this as a bundled dependency of the exporters in order to avoid leaking the unstable JSON types, but i've become less sure.